crates/chia-sdk-driver/src/layers/bulletin_layer.rs

```diff
@@ -0,0 +1,90 @@
+use chia_sdk_types::{
+    puzzles::{IndexWrapperArgs, INDEX_WRAPPER_HASH},
+    Mod,
+};
+use clvm_traits::{FromClvm, MatchByte};
+use clvm_utils::{ToTreeHash, TreeHash};
+use clvmr::{Allocator, NodePtr};
+
+use crate::{DriverError, Layer, Puzzle, SpendContext};
+
+pub type BulletinNonce = (MatchByte<98>, ());
```

Comment:
how does MatchByte work?  Is it a type that matches the byte value of 98?

Reply:
[following on keybase chat]
maybe add a comment here along the lines of 
// BulletinNonce = ('b', ()),